docker/env_file_app_ci

```diff
@@ -88,6 +89,7 @@ TEST_MD5=446c5fbb11b9ce058450555c1c27153c
 TEST_GENERIC=email@example.com
 
 # URLs
+CAPESANDBOX_URL=https://www.capesandbox.com
```

Comment:
please set a fake url here. This file is used by the Github Action

Reply:
will do!